src/frontends/onnx/frontend/src/op/blackmanwindow.cpp

```diff
@@ -27,57 +34,46 @@ OutputVector blackmanwindow(const Node& node) {
 
     // Weights as described in ONNX BlackmanWindow docs
     // https://github.com/onnx/onnx/blob/main/docs/Operators.md#blackmanwindow
-    const auto float_size = std::make_shared<default_opset::Convert>(size, ov::element::f32);
-    const auto a_0 =
-        std::make_shared<default_opset::Constant>(ov::element::f32, ov::Shape(), std::vector<float>{0.42f});
-    const auto a_1 =
-        std::make_shared<default_opset::Constant>(ov::element::f32, ov::Shape(), std::vector<float>{-0.50f});
-    const auto a_2 =
-        std::make_shared<default_opset::Constant>(ov::element::f32, ov::Shape(), std::vector<float>{0.08f});
+    const auto float_size = std::make_shared<v0::Convert>(size, ov::element::f32);
+    const auto a_0 = std::make_shared<v0::Constant>(ov::element::f32, ov::Shape(), std::vector<float>{0.42f});
```

Comment:
fyi, also should work
```suggestion
    const auto a_0 = std::make_shared<v0::Constant>(ov::element::f32, ov::Shape{}, 0.42f);
```

Reply:
Please, tell me about all possible improvements in this and further PRs and I'll apply them while working on api2.0 refactoring. I hope you won't be against I do it in a separate PR for all places.